src/components/modals/WalletListModal.js

```diff
@@ -109,6 +109,27 @@ export function WalletListModal(props: Props) {
     if (wyreClient.isAccountSetup) {
       setBankAccountsMap(await wyreClient.getPaymentMethods())
     }
+
+    setBankAccountsMap({
+      '1': {
+        id: '1',
+        name: 'Wells Fargo 1234',
+        owner: 'string',
+        defaultCurrency: 'USD',
+        status: 'string',
+        supportsDeposit: true,
+        blockchains: { BTC: 'string' }
+      },
+      '2': {
+        id: '2',
+        name: 'Wells Fargo 5678',
+        owner: 'string',
+        defaultCurrency: 'USD',
+        status: 'string',
+        supportsDeposit: true,
+        blockchains: { BTC: 'string' }
+      }
+    })
```

Comment:
This PR doesn't look like it's ready for review. I'll send it back for now. Let's get it ready and then do an OTS to expedite.